proto/Handshake.proto

```diff
@@ -4,11 +4,9 @@ package proto;
 
 import "UiOptions.proto";
 
+// A message which contains either a list of declarations or a list of values for a single module.
 message Handshake {
```

Comment:
Does calling it "Handshake" make sense? If it contains UiValues, do we not expect to receive it every tick?